mongodb/trigger/eventlistener/descriptor.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
are we planning on having multiple mongodb triggers?  If not, maybe we should consider just putting in in "mongodb/trigger/mongodb" . I know it's a little redundant, but a little clearer when using the the api and eventlistener is quite generic.

Reply:
Will modify folder name for trigger from 'eventlistener' to 'mongodb'